### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Added doc for simple-websocket and pyngrok optional packages.

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]
🥇

File: src/taipy/gui/gui.py [hunk 1]
```diff
@@ -89,35 +89,52 @@ class Gui:
...
+
+            The *on_navigate* callback function must return the name of the page the user should be
+            directed to.
         on_exception (Callable): The function that is called an exception occurs on user code.<br/>
             It defaults to the `on_exception()` global function defined in the Python
             application. If there is no such function, exceptions will not trigger
-            anything.
-            TODO: function signature
+            anything.<br/>
+            The signature of the *on_navigate* callback function must be:
```
Comment (FredLL-Avaiga) [inline src/taipy/gui/gui.py]: onçnavigate => on_exception

**Review by FredLL-Avaiga — APPROVED** [review]

**Review by dinhlongviolin1 — APPROVED** [review]
Niceeeeeeeee

File: src/taipy/gui/gui.py (hunk 1 above)
Comment (FabienLelaquais) [reply src/taipy/gui/gui.py]: 👍

**Pull request merged**